tools/run_defacing_script.pl

```diff
@@ -0,0 +1,699 @@
+                       . "all SessionIDs present in the database)";
+
+my @opt_table = (
+    [ "-profile",    "string",  1, \$profile,     $profile_desc      ],
+    [ "-sessionIDs", "string",  1, \$session_ids, $session_ids_desc ],
+    [ "-verbose",    "boolean", 1, \$verbose,     "Be verbose"       ]
+);
+
+my $Help = <<HELP;
+**********************************************************************************
+DEFACE ANATOMICAL SCANS BASED ON SCAN TYPES TO DEFACE IN THE CONFIGURATION MODULE
+**********************************************************************************
+
+This script will run the defacing algorithm on anatomical scan types listed in the
+compute_defaced_images of the imaging pipeline section of the configuration module.
+
+If a list of SessionIDs is provided using the option -sessionIDs, then the defacing
+algorithm will be run restricted to MINC files belonging to those SessionIDs.
+
+If -sessionIDs is not set, the defacing algorithm will be run on MINC files
+belonging to all SessionIDs present in the database.
+
+Documentation: perldoc run_defacing_script.pl
+
+HELP
+
+my $Usage = <<USAGE;
+Usage: $0 [options]
+       $0 -help to list options
+USAGE
+
+&Getopt::Tabular::SetHelp($Help, $Usage);
+&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV)
+    || exit $NeuroDB::ExitCodes::GETOPT_FAILURE;
+
+
+
+## input error checking
+
+if (!$ENV{LORIS_CONFIG}) {
+    print STDERR "\n\tERROR: Environment variable 'LORIS_CONFIG' not set\n\n";
+    exit $NeuroDB::ExitCodes::INVALID_ENVIRONMENT_VAR;
+}
+
+if (!defined $profile || !-e "$ENV{LORIS_CONFIG}/.loris_mri/$profile") {
+    print $Help;
+    print STDERR "$Usage\n\tERROR: You must specify a valid and existing profile.\n\n";
+    exit $NeuroDB::ExitCodes::PROFILE_FAILURE;
+}
+
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
+
+if ( !@Settings::db ) {
+    print STDERR "\n\tERROR: You don't have a \@db setting in the file "
+                 . "$ENV{LORIS_CONFIG}/.loris_mri/$profile \n\n";
+    exit $NeuroDB::ExitCodes::DB_SETTINGS_FAILURE;
+}
+
+
+
+## establish database connection
+
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+print "\n==> Successfully connected to the database \n" if $verbose;
+
+
+
+## get config settings
+
+my $ref_scan_type = &NeuroDB::DBI::getConfigSetting(\$dbh, 'reference_scan_type_for_defacing');
+my $to_deface     = &NeuroDB::DBI::getConfigSetting(\$dbh, 'modalities_to_deface'            );
+my $data_dir      = &NeuroDB::DBI::getConfigSetting(\$dbh, 'dataDirBasepath'                 );
+$data_dir         =~ s/\/$//;  # remove trailing /
+unless ($ref_scan_type && $to_deface) {
+    print STDERR "\n==> ERROR: you need to configure both the "
+                 . "reference_scan_type_for_defacing & modalities_to_deface config "
+                 . "settings in the imaging pipeline section of the Config module.\n"
+                 . "If these configurations are not present, make sure you have run "
```

Comment:
```suggestion
                 . "If these configurations are not present, ensure you have run "
```
more formal than make sure